TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/RuntimeContentCache.php

```diff
@@ -280,18 +280,10 @@ public function evaluateUncached($path, array $contextArray)
     protected function buildCacheIdentifierValues($configuration, $typoScriptPath, $tsObject)
     {
         $cacheIdentifierValues = array();
```

Comment:
This can be much easier without if/else:

```
$objectType = '<TYPO3.TypoScript:GlobalCacheIdentifiers>';
if (isset($configuration['entryIdentifier']['__objectType'])) {
    $objectType=  '<' . $configuration['entryIdentifier']['__objectType'] . '>';
}

return $this->runtime->evaluate($typoScriptPath . '/__meta/cache/entryIdentifier' . $objectType, $tsObject);
```